Documentation/network.md

```diff
@@ -130,10 +130,10 @@ Home Assistant OS will recreate the default connection profile during boot.
 
 ### Enabling WiFi
 
-WiFi is discouraged for reliability reasons. However if you're still like to use WiFi, you can us the `ha network` command to set up WiFi (example for a Raspberry Pi 4, check `ha network info` to check if your board supports WiFi and the name of the WiFi device):
+WiFi is discouraged for reliability reasons. If you would still like to use WiFi, you can us the `ha network` command to set up WiFi (example for a Raspberry Pi 4, check `ha network info` to check if your board supports WiFi and the name of the WiFi device):
 
 ```bash
-ha network update wlan0 --ipv4-method auto --wifi-auth wpa-psk --wifi-mode infrastructure --wifi-ssid MY-SSID --wifi-psk MY_PASS
+ha network update wlan0 --ipv4-method auto --wifi-auth wpa-psk --wifi-mode infrastructure --wifi-ssid "MY-SSID" --wifi-psk MY_PASS
```

Comment:
Ideally we probably also quote the PSK.

```suggestion
ha network update wlan0 --ipv4-method auto --wifi-auth wpa-psk --wifi-mode infrastructure --wifi-ssid "MY-SSID" --wifi-psk "MY-PASS"
```

Reply:
I attempted to use "" on the PSK to test, and the command failed. I think the quotes are not allowed on the PSK (can the PSK contain spaces?)

Reply:
FTR just tested having both with double quote and it's working

Reply:
Tested on my end, quotes are working.